ui/src/shared/utils/fromFlux.ts

```diff
@@ -0,0 +1,262 @@
+import Papa from 'papaparse'
+
+const TO_COLUMN_TYPE = {
+  boolean: 'boolean',
+  unsignedLong: 'number',
+  long: 'number',
+  double: 'number',
+  string: 'string',
+  'dateTime:RFC3339': 'time',
+}
+
+function parseValue(value, columnType) {
```

Comment:
I know your comments make illuminate the fact that the `columnType` is going to be a `js` type, but maybe we can assign an Enum to this?